client/signup/steps/business-details/index.tsx

```diff
@@ -0,0 +1,221 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize, LocalizeProps } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormTextInput from 'components/forms/form-text-input';
+import StepWrapper from 'signup/step-wrapper';
+import { getBusinessAddress, getBusinessName } from 'state/signup/steps/business-details/selectors';
+import { setBusinessAddress, setBusinessName } from 'state/signup/steps/business-details/actions';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { saveSignupStep, submitSignupStep } from 'state/signup/progress/actions';
+import wpcom from 'lib/wp';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	flowName: string;
+	goToNextStep: ( flowName?: string ) => void;
```

Comment:
Where does this come from? Is there a function that's expected to be provided, or do we expect to receive an anonymous function that satisfies this type?